Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM ruby:2.3.7
+FROM ruby:2.3.7-slim
 ENV PHANTOM_JS=2.1.1
 
+# Add basic binaries and clean up the apt cache
+RUN apt-get update \
+  && apt-get install -y bzip2 curl gnupg wget \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
I think `rm -rf /var/lib/apt/lists*` is intended to clean up some disk space and make the Docker container stay small on developers' hard drives.

But we do more `apt-get update` later in the`Dockerfile`, so I don't think this would help much. We will download the contents of this folder again, so this ends up requiring more data to be downloaded, I think. I think the hard drive space being a tad bigger is better than having to download more MB of data from the network.

And we would want to be more consistent with removing this folder's contents after every `apt-get update` to really save hard-drive space.

See: _"what is the purpose of the files in this directory: /var/lib/apt/lists?"_ https://serverfault.com/questions/449726/what-is-the-purpose-of-the-files-in-this-directory-var-lib-apt-lists

Reply:
This would have more impact after the yarn, phantomjs, and nodejs install scripts are done.

But I don't think we are targeting small final image size so much as fast builds. Not sure if rest of team agrees -- Are we going to start using the final built docker image for distributing the app, or in production? In which case having this could help final image size stay small.